br/pkg/restore/log_client/import_retry_test.go

```diff
@@ -609,3 +609,26 @@ func TestPaginateScanLeader(t *testing.T) {
 	})
 	assertRegions(t, collectedRegions, "", "aay", "bba")
 }
+
+func TestRetryRecognizeErrCode(t *testing.T) {
+	waitTime := 1 * time.Millisecond
+	maxWaitTime := 16 * time.Millisecond
+	ctx := context.Background()
+	inner := 0
+	outer := 0
+	utils.WithRetry(ctx, func() error {
+		e := utils.WithRetry(ctx, func() error {
+			inner++
+			e := status.Error(codes.Unavailable, "the connection to TiKV has been cut by a neko, meow :3")
+			if e != nil {
+				return errors.Trace(e)
+			}
+			return nil
+		}, utils.NewBackoffer(10, waitTime, maxWaitTime, utils.NewErrorContext("download sst", 3)))
```

Comment:
I guess `waitTime` and `maxWaitTime` can both be zero?

Reply:
maybe, but it harms nothing.